rust/chaosymmetry/src/main.rs

```diff
@@ -179,36 +195,56 @@ impl ApplicationHandler for App {
                 is_synthetic: false,
                 ..
             } => {
-                if event.logical_key == Key::Character("+".into()) && event.state.is_pressed() {
-                    self.render.scale *= ZOOM_FACTOR;
-                } else if event.logical_key == Key::Character("-".into())
-                    && event.state.is_pressed()
+                if event.state.is_pressed()
+                    // Since "+" requires pressing "Shift", this would otherwise reset the multiplier to "1" before zooming in
+                    && event.logical_key != Key::Named(winit::keyboard::NamedKey::Shift)
```

Comment:
Dit kunnen we denk ik nog wel wat refactoren:
- Hier is een example van hoe winit zelf het doet: https://github.com/rust-windowing/winit/blob/master/examples/application.rs#L461-L476
- Deze method kan van pas komen: https://docs.rs/winit/latest/winit/platform/modifier_supplement/trait.KeyEventExtModifierSupplement.html#tymethod.text_with_all_modifiers